store/timescale.go

```diff
@@ -0,0 +1,37 @@
+package store
+
+import (
+	"database/sql"
+
+	"github.com/jackc/pgx/stdlib"
+	"github.com/pkg/errors"
+	"github.com/replicatedcom/saaskit/tracing/datadog"
+)
+
+var TimescaleDB *sql.DB
+
+type TimescaleOpts struct {
+	URI string
+}
+
+func InitTimescale(opts TimescaleOpts) error {
+	if opts.URI == "" {
+		return errors.New("Timescale URI is not set")
+	}
+
+	RegisterDatadogDriver("pgx", &stdlib.Driver{}, "timescale")
+	db, err := datadog.OpenSQL("pgx", opts.URI)
+	if err != nil {
+		return errors.Wrap(err, "open pgx")
+	}
+
+	TimescaleDB = db
```

Comment:
Before this assignment, should we try and ping the database via `db.QueryRow("SELECT 1")`.